from app.domain.base.base_exception import BaseDomainError


class UserDeletedError(BaseDomainError):
    pass
